essentials/src/types.rs

```diff
@@ -35,14 +41,25 @@ pub type SubxtCall = runtime::RuntimeCall;
 
 pub type ClaimQueue = BTreeMap<u32, VecDeque<Option<ParasEntry>>>;
 
+// TODO: Take it from runtime types v5
+/// How a free core is scheduled to be assigned.
+pub struct CoreAssignment {
+	/// The core that is assigned.
+	pub core: CoreIndex,
+	/// The unique ID of the para that is assigned to the core.
+	pub para_id: Id,
+	/// The kind of the assignment.
+	pub kind: AssignmentKind,
+}
+
 // TODO: Take it from runtime types v5
 /// Polkadot v5 ParasEntry type
 #[derive(Debug)]
 pub struct ParasEntry {
 	/// The `Assignment`
 	pub assignment: Assignment,
-	/// Number of times this has been retried.
-	pub retries: u32,
+	/// The number of times the entry has timed out in availability.
+	pub availability_timeouts: u32,
```

Comment:
The meaning of these fields seems to be quite similar, so we should probably use it for the both elements returned from the runtime.

Reply:
This change was made to give the field a more descriptive name due to ambiguity.